### ISSUE TITLE
Issue with hotspot image connector and latest develop

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#824]
This shows the new hotspot icons when adding/editing hotspots but I can't add hotspots and so can't fully test the page either.

**Comment by torinfo** [comment]
I addressed these concerns, and also the concerns mentioned in #823 

- I created a new hotspot editor 
- hotspot connector and hotspot image support flexible hotspots and rotated rectangles
- annotated diagram and drag drop hotspots are limited to rectangles

When you edit existing hotspots in hotspot image, these are restricted to ractangles. Creaet a new hotspot to create a flexible hotspot

Questions:
 - There is some help text below the image. Are these adequate? Sensible and correct English?
 - I introduced a 'Reset' icon, to restart drawing a polygon (or rectangle). Is the icon crear enough, and is the term 'Reset'clear enough.
